credoai/reporting/dataset_base.py

```diff
@@ -47,134 +45,116 @@ def _plot_balance_metrics(self):
+
+            # Generate sample balance barplots
+            results = results_all["sample_balance"]
+            df = pd.DataFrame(results)
+            sensitive_feature_name = list(df.drop(["count", "percentage"], axis=1).columns)[
+                0
+            ]
+            ax = sns.barplot(
+                x="count",
+                y=sensitive_feature_name,
+                data=df,
+                palette=plot_utils.credo_diverging_palette(1),
+                ax=axes[0],
+            )
+            f.patch.set_facecolor("white")
+            sns.despine()
+            ax.set_title("Data balance across " + sensitive_feature_name + " subgroups")
+            ax.set_xlabel("Number of data samples")
+            ax.set_ylabel("")
+
+            # Generate label balance barplots
+            results = results_all["label_balance"]
+            df = pd.DataFrame(results)
+            label_name = list(df.drop([sensitive_feature_name, "count"], axis=1).columns)[0]
+
+            num_classes = df[label_name].nunique()
+            ax = sns.barplot(
+                x="count",
+                y=sensitive_feature_name,
+                hue=label_name,
+                data=df,
+                palette=plot_utils.credo_diverging_palette(num_classes),
+                alpha=1,
+                ax=axes[1],
+            )
+            f.patch.set_facecolor("white")
+            sns.despine()
+            ax.set_title(
+                "Data balance across "
+                + sensitive_feature_name
+                + " subgroups and label values"
+            )
+            ax.set_xlabel("Number of data samples")
+            ax.set_ylabel("")
+            ax.get_legend().set_visible(False)
+
+            # Generate parity metrics barplots
+            metric_keys = ['demographic_parity_difference',
+                        'demographic_parity_ratio']
+
+            lst = []
+            for metric in metric_keys:
+                temp = pd.DataFrame(results_all[metric])
+                temp["metric"] = metric.replace("_", " ")
+                lst.append(temp)
+
+            df = pd.concat(lst)
+            ax = sns.barplot(
+                x="value",
+                y="metric",
+                hue=label_name,
+                data=df,
+                palette=plot_utils.credo_diverging_palette(num_classes),
+                ax=axes[2],
+            )
+            f.patch.set_facecolor("white")
+            sns.despine()
+            plt.title("Parity metrics for different preferred label value possibilities")
+            plt.xlabel("Value")
+            plt.ylabel("")
+            plt.legend(
+                bbox_to_anchor=(1.2, 0.5), 
+                loc="center",
+                frameon=False,
+                ncol=num_classes,
+                title=label_name
+            )
+            ax.legend_.set_title(label_name)
+        self.figs.append(f)
 
     def _plot_group_diff(self):
```

Comment:
This and the mutual information plots, which are both bar plots and would look better in landscape, are now plotted in portrait aspect ratio. 
![image](https://user-images.githubusercontent.com/92744575/156009847-926f67d9-2be7-4ad1-9051-4939b150e79c.png)

Reply:
Good point - fixed